lib/gui/lvgl/Kconfig.objects

```diff
@@ -23,6 +24,117 @@ choice
 
 endchoice
 
+config LVGL_OBJ_ARC
+	bool "Arc Object"
+	help
+	  Enable arc object support
+
+config LVGL_OBJ_BAR
+	bool "Bar object"
+	help
+	  Enable bar object support
+
+config LVGL_OBJ_BTN
```

Comment:
I don't object to the rename to align with LVGL, but would strongly prefer that the renaming be decoupled (in a separate commit from) from the re-ordering.  I agree with @jfischer-phytec-iot that as-is it makes it very hard to see what options have changed.